client/pkg/config/authconfig.go

```diff
@@ -7,6 +7,7 @@ type AuthConfig struct {
 	ServerPassword string `config:"serverPassword"`
 	Username       string `config:"username"`
 	Password       string `config:"password"`
+	BasicAuth      bool   `config:"useBasicAuth"`
```

Comment:
Should this be "HasBasicAuth" or "UseBasicAuth" ?

Reply:
The attribute is just basic auth, the getter is UseBasicAuth, I think keeping it this way is fine. Thanks for the suggestion.